### PR TITLE
Preview in ranking filter component

### PR DESCRIPTION
**Pull request by xMort**
JIRA: BB-2505

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by xMort** [comment]
extended test

File: libs/sdk-ui/src/base/localization/bundles/en-US.json [hunk 1]
```diff
@@ -971,5 +971,30 @@
...
+        "limit": 0
+    },
+    "rankingFilter.preview.top_with_attribute": {
+        "value": "Top {value} out of <strong>{attribute}</strong> based on <strong>{measure}</strong>",
+        "comment": "Do not translate placeholder {value}, {attribute} or HTML markup enclosed in <>, and {measure}. Title describing ranking direction and number of values. Example: 'Top 3 out of Product based on Revenue'",
+        "limit": 0
+    },
+    "rankingFilter.preview.bottom_with_attribute": {
+        "value": "Bottom {value} out of <strong>{attribute}</strong> based on <strong>{measure}</strong>",
+        "comment": "Do not translate placeholder {value}, {attribute}, and {measure} or HTML markup enclosed in <>. Title describing ranking direction and number of values. Example: 'Top 5 out of Product based on Revenue'",
```
Comment (scavnickyj) [inline libs/sdk-ui/src/base/localization/bundles/en-US.json:997]: `Example: 'Bottom 5 out of Product based on Revenue'`

I think the middle sentence of last 4 translations should be consistent and maybe mention all placeholders.

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/873/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/342/ : SUCCESS in 12m 39s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/872/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2458/ : FAILURE in 16m 23s

**Comment by xMort** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/877/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2465/ : SUCCESS in 25m 35s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/878/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/343/ : SUCCESS in 31m 46s

**Review by scavnickyj — APPROVED** [review]

**Comment by scavnickyj** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/355/ : SUCCESS in 31s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/611/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2466/ : SUCCESS in 14m 56s

**Pull request merged**